compiler/src/Language/Bond/Codegen/Cpp/Apply_cpp.hs

```diff
@@ -21,15 +20,13 @@ apply_cpp protocols cpp file _imports declarations = ("_apply.cpp", [lt|
 #include "#{file}_apply.h"
 #include "#{file}_reflection.h"
 
-#{CPP.openNamespace cpp}
-    #{newlineSepEnd 1 (applyOverloads protocols cpp attr body) declarations}
-#{CPP.closeNamespace cpp}
+namespace bond
+{
+    #{newlineSepEnd 1 (applyOverloads protocols cpp attr extern) declarations}
+} // namespace bond
 |])
   where
-    body = [lt|
-    {
-        return ::bond::Apply<>(transform, value);
-    }|]
-
     attr = [lt||]
 
+    extern = [lt|template|]
```

Comment:
Wouldn't it be more appropriate to have the literal 'template' appear directly in the applyOverloads function and have extern defined as 'extern = [lt||]' here and as 'extern = [lt|extern|}' here?